README.md

```diff
@@ -79,11 +79,17 @@ Clone this repository to your desired folder: (https://github.com/HERMON-1995/Le
   git clone [https://github.com/HERMON-1995/Leaderboard-list-app-project.git]
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/223367028-645beeab-8d40-4dc8-8750-33fa4b10dddc.png)

**[Optional]**

- [ ] If i remember here should be a link to your issues page :smile: which is missing :thinking: Kindly put it here to make your readme file descriptive and professional :blush: